lib/data/fake_model/fake_product_repository.dart

```diff
@@ -149,7 +149,8 @@ class FakeProductRepository extends ProductRepository with FavoritesRepository {
       SortRules sortRules = const SortRules(),
       FilterRules filterRules}) async {
     return (await _getFavorites())
-        .map((value) => FavoriteProduct(value, HashMap()));
+        .map((value) => FavoriteProduct(value, HashMap()))
+        .toList();
```

Comment:
Add `toList()` to return favorite items. Earlier it was not returning empty results.

Reply:
ok